fury/actor.py

```diff
@@ -65,15 +69,22 @@ def actor_from_primitive(
         material and properties
 
     """
-    res = fp.repeat_primitive(
-        vertices,
-        faces,
-        directions=directions,
-        centers=centers,
-        colors=colors,
-        scales=scales,
-    )
-    big_vertices, big_faces, big_colors, _ = res
+
+    if repeat_primitive:
+        res = fp.repeat_primitive(
+            vertices,
+            faces,
+            directions=directions,
+            centers=centers,
+            colors=colors,
+            scales=scales,
+        )
```

Comment:
```suggestion
        res = fp.repeat_primitive(
            vertices,
            faces,
            centers,
            directions=directions,
            colors=colors,
            scales=scales,
        )
```

This allows everyone by looking why the if repeat_primitive is required.